source/Octopus.Tentacle.Client.Tests/RpcCallRetryHandlerFixture.cs

```diff
@@ -687,5 +688,194 @@ private TimeSpan GetMinTimeoutDuration(RpcCallRetryHandler handler)
         {
             return handler.RetryTimeout - handler.RetryIfRemainingDurationAtLeast - retryBackoffBuffer;
         }
+
+        [Test]
+        public async Task MinimumAttempts_WhenSetTo1_DoesNotRetryAfterTimeoutExceeded()
```

Comment:
Is this test is testing what happens after we timeout, where does it do the timeout? In other tests, we delay in the lambda (that increments `callCount`). Am I missing something?

Reply:
fixed sleeping in the lambda